pkg/config/config.go

```diff
@@ -277,6 +278,15 @@ func New() (*Config, error) {
 		yaml.Unmarshal(b, c)
 	}
 
+	if c.Settings.ClusterName == "+google_metadata+" {
```

Comment:
Can we use something like `{{ google_metada }}` instead? Looks more like go templating

Reply:
Using {{}} would suggest that Go templating is actually supported. If it is not, then this only leads to confusion, I think.

Would `$GOOGLE_METADATA` or `google-metadata:///cluster` make more sense to you?

Reply:
Sounds good to me.